obs/preinstallimage-bios.sh.in

```diff
@@ -822,6 +822,8 @@ install -m 0755 /usr/share/fty/scripts/ethtool-static-nolink /etc/network/if-pre
 install -m 0755 /usr/share/fty/scripts/ifupdown-force /etc/ifplugd/action.d/ifupdown-force
 install -m 0755 /usr/share/fty/scripts/udhcpc-override.sh /usr/local/sbin/udhcpc
 echo '[ -s /usr/share/fty/scripts/udhcpc-hook.sh ] && . /usr/share/fty/scripts/udhcpc-hook.sh' >> /etc/udhcpc/default.script
+# Get it into our default PATH:
+ln -fsr /usr/libexec/fty/fty-hostname-setup /usr/local/sbin/fty-hostname-setup
```

Comment:
`/usr/libexec/fty` is already in the PATH, seemingly thanks to `/etc/profiled.d/fty_path.sh`.

Reply:
This is a path for interactive shells, not necessarily for system services.